proto/stride/stakeibc/params.proto

```diff
@@ -34,4 +34,6 @@ message Params {
   uint64 ibc_transfer_timeout_nanos = 16;
   uint64 safety_num_validators = 17;
   uint64 safety_max_slash_percent = 18;
+  map<string, string> max_redemption_rates = 19; // maxrrs = {chainidA: "1.5", chainidB: "2.0",…}
```

Comment:
Could using a map here potentially lead to a non-determinism error?

Reply:
What sort of non-determinism error? When it's not set on map, it's using default value.

Reply:
I believe just storing a map can cause a consensus error because maps are non-determinstic in go - meaning different validators could attempt to store the same map, but the serialized representation of it could be slightly differently. 
You can get around this by just creating another prefix store that acts as a map

Reply:
@jstr1121 following up on this, we should use a prefix store that acts as a map. Here's an example https://github.com/Stride-Labs/stride/pull/556/commits/aa0af4927db96778d7565210a1a12f2946f900d3

Reply:
We'd like to get this PR into the v7 happening next week. Do you think we can make that happen?

Reply:
I think we wanted to put this in Params - otherwise, we will need to have a separate proposal to change these values.